Wikipedia/UI-V5/WMFAppViewController.m

```diff
@@ -35,6 +55,8 @@ @interface WMFAppViewController ()<UITabBarControllerDelegate>
 
 @property (nonatomic, strong) SessionSingleton* session;
 
+@property (nonatomic, strong) WMFNavigationTransitionController* navigationTransitionController;
```

Comment:
Whats the purpose of the navigation controller here? Is this the list transition?